python/sglang/srt/conversation.py

```diff
@@ -648,6 +649,20 @@ def generate_chat_conv(
     )
 )
 
+register_conv_template(
+    Conversation(
+        name="devstral",
+        system_template="[SYSTEM_PROMPT]\n{system_message}\n[/SYSTEM_PROMPT]\n\n",
+        system_message=read_system_prompt_from_file("mistralai/Devstral-Small-2505"),
```

Comment:
The implementation of `read_system_prompt_from_file` can be a bit tricky. But I'm completely fine with merging this first and unblocking the usage.